gui_qt/app.py

```
@@ -1584,6 +1678,35 @@ def _on_finished(self, exit_code: int):
+        if self._active_command == "bootstrap_work":
+            summary = summarize_work_bootstrap_output(
+                "\n".join(self._work_bootstrap_output_lines),
+                exit_code,
+            )
+            game_root_update_failed = False
+            if summary.status == "ready" and summary.work_dir:
+                try:
+                    self.state.set_game_root(summary.work_dir)
+                    self._refresh_project_label()
+                except ValueError as exc:
+                    self._append_log(f"更新 translator_config.json 失败：{exc}")
```

Ruff (0.15.18)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)